mmv1/third_party/terraform/services/storage/resource_storage_bucket_test.go.erb

```diff
@@ -76,12 +78,13 @@ func TestAccStorageBucket_basicWithAutoclass(t *testing.T) {
 				ImportStateVerify:       true,
 				ImportStateVerifyIgnore: []string{"force_destroy"},
 			},
-			// Autoclass is ForceNew, so this destroys & recreates, but does test the explicitly disabled config
 			{
-				Config: testAccStorageBucket_basicWithAutoclass(bucketName, false),
+				Config: testAccStorageBucket_basicWithAutoclass_update(bucketName, true),
 				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr(
-						"google_storage_bucket.bucket", "force_destroy", "false"),
+					testAccCheckStorageBucketExists(
+						t, "google_storage_bucket.bucket", bucketName, &updated),
+					testAccCheckStorageBucketWasUpdated(&updated, &bucket),
+					testAccCheckAutoclassTerminalStorageClass("ARCHIVE", &updated),
```

Comment:
Generally speaking, checks are only needed if the thing that you're testing is not returned by the API - for example, force_destroy is a Terraform-side flag, not an API field. Fields like terminal_storage_class will generally get checked during the ImportState step. Just want to double-check that this needs to be handled as a check?

Reply:
I set the field as `Computed` and wanted to be really clear that it was being set.

Reply:
I did this more for my own verification while I was testing, so I'm just going to delete it.